docs/conf.py

```diff
@@ -220,18 +220,11 @@ def notebooks_to_rst(app):
         from nbconvert.exporters import RSTExporter
         from nbformat import NotebookNode
     except ImportError:
-        try:
-            from IPython.nbconvert.nbconvertapp import NbConvertApp
-            from IPython.nbconvert.writers import FilesWriter
-            from IPython.nbconvert.preprocessors import Preprocessor, ExecutePreprocessor, execute
-            from IPython.nbconvert.exporters import RSTExporter
-            from IPython.nbformat import NotebookNode
-        except ImportError:
-            raise ImportError('Failed to find Jupyter or IPython. Cannot build '
-                              'the notebooks embedded in the docs. Proceeding '
-                              'the rest of the doc build, but additional '
-                              'warnings are likely.')
-            return
+        raise ImportError('Failed to find nbconvert. Cannot build '
```

Comment:
Actually you can remove the whole function, as we don't have notebooks any more in this repo as don't include them either in the docs from ``photutils-datasets``